bucket/snaptimer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.1",
+    "description": "SnapTimer - Free Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": "Freeware",
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "227df1414eefda1d98249d6e96465266a9b6517e29808e0eb57f3eca8875d23b",
+    "extract_dir": "SnapTimer",
+    "bin": "SnapTimer.exe",
+    "shortcuts": [
+        [
+            "SnapTimer.exe",
+            "SnapTimer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://dan.hersam.com/software/snaptimer/index.php",
+        "regex": "<b>Version:</b> (\\d+\\.\\d+)"
+    },
+    "autoupdate": {
```

Comment:
Add hash extraction

![image](https://user-images.githubusercontent.com/13260377/97573440-1331bb00-19ea-11eb-87b5-fe37bd597bce.png)

Reply:
Implement hash extraction into autoupdate and then run checkver to generate hash, instead of adding it manually.